pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+**Related issues**
+*Add links to related issues here. Use [keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) if you want an issue to be automatically closed when the PR is merged.*
+
+**Description**
+*If the changes in the PR are not sufficiently explained by the related issues and commit messages, add a description here.*
+
+**Merge checklist**
+*Complete the checklist before requesting a review.*
+- [ ] [Log your changes](https://ocds-standard-development-handbook.readthedocs.io/en/latest/standard/contributing.html#logging-changes)
+
+If there are changes to `project-schema.json` or `project-package-schema.json`:
```

Comment:
Do we need a bullet here about updating the `update` function in `manage.py`, if the changes overwrite a field shared with OCDS for PPPs?

Reply:
Ah, good spot. To reduce overhead on individual PRs, I wonder if it might be easier to add some instructions on making those updates to: https://ocds-standard-development-handbook.readthedocs.io/en/latest/profiles/infrastructure.html#copy-schema-and-codelists-from-ocds-for-ppps?

Reply:
Sure, I figure changes to fields imported from OCDS are done deliberately, so we won't end up in a situation where we need to consider reverting a change.

Reply:
Handbook PR: https://github.com/open-contracting/standard-development-handbook/pull/273